### DISCUSSION TITLE
Questions

### DISCUSSION THREAD

**Discussion by WSrujan** [marioparaschiv/nitro-sniper#176] [Q&A]
How many giveaway bots are supported and which?
Should i update my sniper (1 July 2021)?
is there limit on number of credits claimed ( i mean on discord side more = ban)?
what is optimal Server number and tokens?
Will this repo be updated?

**Comment by nichevulcan20** [comment]
All giveaway bots are supported except the bots like plasma. Saya etc which requires joining the servers. cuz the sniper can only react to the giveaways, it cant do the tasks given by the bot.

Yes, if you are having bugs or errors you can. because updates fix the bugs!

There is no limit to account credit. but getting too many nitros in very little time is sus.. 

Joining too many servers is not good at all. There is a trick to find out which server drops nitros and which doesn't. Go to the server and search for `discord.gift/` and u will get results. If there are results then be on the server. If no, then leave the server. 

idk
